src/app/state/index.ts

```diff
@@ -0,0 +1,9 @@
+import { movieReducer } from './movies';
+import { MovieState } from './movies/movies-state';
+
+export interface AppState {
+  movie: MovieState;
+}
+export const reducers = {
+  movies: movieReducer,
```

Comment:
możnaby to zmienić z movies na movie - masz movie: MovieState, to trzymałbym się liczby pojedyńczej. Kosmetyka